src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/AcknowledgeClaimCallbackHandler.java

```diff
@@ -276,17 +262,14 @@ && respondent2HasSameLegalRep(caseData)) {
 
         } else if (caseData.getAddRespondent2() != null && caseData.getAddRespondent2().equals(YES)
```

Comment:
Could use the opportunity to refactor a bit like this conditional could be extracted to a method for better readability